### ISSUE TITLE
when an address or location is not in mesa, we need to tell the user

### ISSUE THREAD

**Issue by tbuckl** [codeforamerica/MuniciPal#90]
this used to work, but i can't recall where it fell off.

**Comment by michaelhwan** [comment]
@techieshark @tombuckley @wendyfongsf was the original idea to notify the user with an alert?

**Comment by techieshark** [comment]
@michaelhwan some sort of visual notification that we cant look up things outside the city

try dragging the map marker outside Mesa or typing in 'scottsdale' in the address field. nothing happens. better if we had feedback near their action - eg if the map marker popup text says 'This isnt in Mesa' when dragged out, and if error msg near address input says 'Only Mesa addresses will work.'

![image](https://cloud.githubusercontent.com/assets/1072292/19246162/7a924262-8f81-11e6-991f-51eb66ccc16b.png)

![image](https://cloud.githubusercontent.com/assets/1072292/19246356/6db10ee2-8f82-11e6-8999-f1d7fade23d2.png)

**Comment by michaelhwan** [comment]
@techieshark I think real world users will type out their address instead of dragging it to their home address. Pinning your address requires incredible precision in Google Maps.

**Comment by techieshark** [comment]
@michaelhwan yeah dragging is more of a "I wonder what's happening in this part of town" kind of feature rather than a way to provide your own address.
